jaeger/cmd/install.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/linkerd/linkerd2/jaeger/flag"
+	jaeger "github.com/linkerd/linkerd2/jaeger/values"
+	"github.com/linkerd/linkerd2/pkg/charts"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"k8s.io/helm/pkg/chartutil"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	templatesJaeger = []string{
+		"templates/namespace.yaml",
+		"templates/tracing-rbac.yaml",
+		"templates/tracing.yaml",
+	}
+)
+
+func newCmdInstall() *cobra.Command {
+	var skipChecks bool
+
+	values, err := jaeger.NewValues()
+	if err != nil {
+		fmt.Fprint(os.Stderr, err.Error())
+		os.Exit(1)
+	}
+
+	flags, installOnlyFlagSet := makeInstallFlags(values)
+
+	cmd := &cobra.Command{
+		Use:   "install [flags]",
+		Args:  cobra.NoArgs,
+		Short: "Output Kubernetes resources to install jaeger extension",
+		Long:  `Output Kubernetes resources to install jaeger extension.`,
+		Example: `  # Default install.
+  linkerd jaeger install | kubectl apply -f -
+
+  # Install Linkerd into a non-default namespace.
+  linkerd jaeger install --namespace linkerdtest | kubectl apply -f -`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if !skipChecks {
+				// TODO: Add Checks for checking if linkerd exists
+				// Also check for jaeger
```

Comment:
I don't think we want to error if jaeger already exists.  We want to simply overwrite the previous installation, since this is the flow for updates.